packages/ui/src/components/editor/panels/Files/icon/index.tsx

```diff
@@ -90,7 +90,7 @@ export const FileIcon = ({
           alt=""
         />
       ) : FallbackIcon ? (
-        <FallbackIcon className={`${color}`} />
+        <FallbackIcon className={`${color} h-full w-full`} />
```

Comment:
make the fallback occupy 85-90% instead of full
![image](https://github.com/user-attachments/assets/db651b36-ed7c-4cc4-b263-284106013cb6)

Reply:
I added some padding around